<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="enabled">
   <bool>false</bool>
  </property>
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>350</width>
    <height>650</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>350</width>
    <height>650</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>350</width>
    <height>650</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <property name="styleSheet">
   <string notr="true">background-color: rgb(255, 255, 255);</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <property name="sizePolicy">
    <sizepolicy hsizetype="Maximum" vsizetype="Maximum">
     <horstretch>0</horstretch>
     <verstretch>0</verstretch>
    </sizepolicy>
   </property>
   <property name="layoutDirection">
    <enum>Qt::RightToLeft</enum>
   </property>
   <layout class="QVBoxLayout" name="verticalLayout">
    <property name="spacing">
     <number>0</number>
    </property>
    <item>
     <widget class="QTextEdit" name="Display">
      <property name="minimumSize">
       <size>
        <width>0</width>
        <height>150</height>
       </size>
      </property>
      <property name="maximumSize">
       <size>
        <width>16777215</width>
        <height>150</height>
       </size>
      </property>
      <property name="font">
       <font>
        <family>Noto Sans</family>
        <pointsize>40</pointsize>
       </font>
      </property>
      <property name="layoutDirection">
       <enum>Qt::RightToLeft</enum>
      </property>
      <property name="styleSheet">
       <string notr="true">border: 0px;</string>
      </property>
      <property name="readOnly">
       <bool>true</bool>
      </property>
      <property name="markdown">
       <string>0

</string>
      </property>
      <property name="html">
       <string>&lt;!DOCTYPE HTML PUBLIC &quot;-//W3C//DTD HTML 4.0//EN&quot; &quot;http://www.w3.org/TR/REC-html40/strict.dtd&quot;&gt;
&lt;html&gt;&lt;head&gt;&lt;meta name=&quot;qrichtext&quot; content=&quot;1&quot; /&gt;&lt;meta charset=&quot;utf-8&quot; /&gt;&lt;style type=&quot;text/css&quot;&gt;
p, li { white-space: pre-wrap; }
hr { height: 1px; border-width: 0; }
li.unchecked::marker { content: &quot;\2610&quot;; }
li.checked::marker { content: &quot;\2612&quot;; }
&lt;/style&gt;&lt;/head&gt;&lt;body style=&quot; font-family:'Noto Sans'; font-size:40pt; font-weight:400; font-style:normal;&quot;&gt;
&lt;p align=&quot;right&quot; style=&quot; margin-top:26px; margin-bottom:26px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px;&quot;&gt;&lt;span style=&quot; color:#000000;&quot;&gt;0&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
      </property>
      <property name="placeholderText">
       <string>0</string>
      </property>
     </widget>
    </item>
    <item>
     <spacer name="verticalSpacer">
      <property name="orientation">
       <enum>Qt::Vertical</enum>
      </property>
      <property name="sizeHint" stdset="0">
       <size>
        <width>20</width>
        <height>40</height>
       </size>
      </property>
     </spacer>
    </item>
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout_6">
      <property name="spacing">
       <number>12</number>
      </property>
      <property name="leftMargin">
       <number>23</number>
      </property>
      <property name="topMargin">
       <number>10</number>
      </property>
      <property name="rightMargin">
       <number>23</number>
      </property>
      <property name="bottomMargin">
       <number>12</number>
      </property>
      <item>
       <layout class="QVBoxLayout" name="verticalLayout_9" stretch="0,0,0,0">
        <property name="spacing">
         <number>12</number>
        </property>
        <property name="topMargin">
         <number>0</number>
        </property>
        <item>
         <widget class="QPushButton" name="ButtonCe">
          <property name="font">
           <font>
            <family>Noto Sans</family>
            <pointsize>15</pointsize>
           </font>
          </property>
          <property name="layoutDirection">
           <enum>Qt::RightToLeft</enum>
          </property>
          <property name="styleSheet">
           <string notr="true">QPushButton{
	background-color: #FF4BAC;
 	border-style: solid;
 	border-radius:30px;
	max-width:60px;
	max-height:60px;
	min-width:60px;
	min-height:60px;
	color: white;

}</string>
          </property>
          <property name="text">
           <string>CE</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="ButtonMul">
          <property name="font">
           <font>
            <family>Noto Sans</family>
            <pointsize>20</pointsize>
           </font>
          </property>
          <property name="layoutDirection">
           <enum>Qt::RightToLeft</enum>
          </property>
          <property name="styleSheet">
           <string notr="true">QPushButton{
	background-color: #FF4BAC;
 	border-style: solid;
 	border-radius:30px;
	max-width:60px;
	max-height:60px;
	min-width:60px;
	min-height:60px;
	color: white;

}</string>
          </property>
          <property name="text">
           <string>⨯</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="Button1_2">
          <property name="minimumSize">
           <size>
            <width>60</width>
            <height>60</height>
           </size>
          </property>
          <property name="font">
           <font>
            <family>Noto Sans</family>
            <pointsize>20</pointsize>
           </font>
          </property>
          <property name="cursor">
           <cursorShape>PointingHandCursor</cursorShape>
          </property>
          <property name="styleSheet">
           <string notr="true">QPushButton{
	background-color: #FF4BAC;
 	border-style: solid;
 	border-radius:30px;
	max-width:60px;
	max-height:60px;
	min-width:60px;
	min-height:60px;
	color: white;

}</string>
          </property>
          <property name="text">
           <string>-</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="ButtonAdd">
          <property name="font">
           <font>
            <family>Noto Sans</family>
            <pointsize>20</pointsize>
           </font>
          </property>
          <property name="layoutDirection">
           <enum>Qt::RightToLeft</enum>
          </property>
          <property name="styleSheet">
           <string notr="true">QPushButton{
	background-color: #FF4BAC;
 	border-style: solid;
 	border-radius:30px;
	max-width:60px;
	max-height:60px;
	min-width:60px;
	min-height:60px;
	color: white;

}</string>
          </property>
          <property name="text">
           <string>+</string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item>
       <layout class="QVBoxLayout" name="verticalLayout_8">
        <property name="spacing">
         <number>12</number>
        </property>
        <item>
         <widget class="QPushButton" name="ButtonMod">
          <property name="font">
           <font>
            <family>Noto Sans</family>
            <pointsize>16</pointsize>
           </font>
          </property>
          <property name="layoutDirection">
           <enum>Qt::RightToLeft</enum>
          </property>
          <property name="styleSheet">
           <string notr="true">QPushButton{
	background-color: #FF4BAC;
 	border-style: solid;
 	border-radius:30px;
	max-width:60px;
	max-height:60px;
	min-width:60px;
	min-height:60px;
	color: white;

}</string>
          </property>
          <property name="text">
           <string>mod</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="ButtonDiv">
          <property name="font">
           <font>
            <family>Noto Sans</family>
            <pointsize>20</pointsize>
           </font>
          </property>
          <property name="layoutDirection">
           <enum>Qt::RightToLeft</enum>
          </property>
          <property name="styleSheet">
           <string notr="true">QPushButton{
	background-color: #FF4BAC;
 	border-style: solid;
 	border-radius:30px;
	max-width:60px;
	max-height:60px;
	min-width:60px;
	min-height:60px;
	color: white;

}</string>
          </property>
          <property name="text">
           <string>÷</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="Button9">
          <property name="font">
           <font>
            <family>Noto Sans</family>
            <pointsize>20</pointsize>
           </font>
          </property>
          <property name="layoutDirection">
           <enum>Qt::RightToLeft</enum>
          </property>
          <property name="styleSheet">
           <string notr="true">QPushButton{
	background-color: white;
	border: 1px;
 	border-style: solid;
	border-color: rgb(255, 212, 246);
	color: #82005D;

 	border-radius:30px;
	max-width:60px;
	max-height:60px;
	min-width:60px;
	min-height:60px;
	

}</string>
          </property>
          <property name="text">
           <string>9</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="Button6">
          <property name="font">
           <font>
            <family>Noto Sans</family>
            <pointsize>20</pointsize>
           </font>
          </property>
          <property name="layoutDirection">
           <enum>Qt::RightToLeft</enum>
          </property>
          <property name="styleSheet">
           <string notr="true">QPushButton{
	background-color: white;
	border: 1px;
 	border-style: solid;
	border-color: rgb(255, 212, 246);
	color: #82005D;

 	border-radius:30px;
	max-width:60px;
	max-height:60px;
	min-width:60px;
	min-height:60px;
	

}</string>
          </property>
          <property name="text">
           <string>6</string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item>
       <layout class="QVBoxLayout" name="verticalLayout_7">
        <property name="spacing">
         <number>12</number>
        </property>
        <item>
         <widget class="QPushButton" name="ButtonRoot">
          <property name="font">
           <font>
            <family>Noto Sans</family>
            <pointsize>20</pointsize>
           </font>
          </property>
          <property name="layoutDirection">
           <enum>Qt::RightToLeft</enum>
          </property>
          <property name="styleSheet">
           <string notr="true">QPushButton{
	background-color: #FF4BAC;
 	border-style: solid;
 	border-radius:30px;
	max-width:60px;
	max-height:60px;
	min-width:60px;
	min-height:60px;
	color: white;

}</string>
          </property>
          <property name="text">
           <string/>
          </property>
          <property name="icon">
           <iconset>
            <disabledon>:/icons/root_n.png</disabledon>
            <activeon>:/icons/root_n.png</activeon>
           </iconset>
          </property>
          <property name="iconSize">
           <size>
            <width>35</width>
            <height>25</height>
           </size>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="ButtonSqaureRoot">
          <property name="font">
           <font>
            <family>Noto Sans</family>
            <pointsize>20</pointsize>
           </font>
          </property>
          <property name="layoutDirection">
           <enum>Qt::RightToLeft</enum>
          </property>
          <property name="styleSheet">
           <string notr="true">QPushButton{
	background-color: #FF4BAC;
 	border-style: solid;
 	border-radius:30px;
	max-width:60px;
	max-height:60px;
	min-width:60px;
	min-height:60px;
	color: white;

}</string>
          </property>
          <property name="text">
           <string/>
          </property>
          <property name="icon">
           <iconset>
            <disabledon>:/icons/sqaure_root.png</disabledon>
            <activeon>:/icons/sqaure_root.png</activeon>
           </iconset>
          </property>
          <property name="iconSize">
           <size>
            <width>26</width>
            <height>20</height>
           </size>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="Button8">
          <property name="font">
           <font>
            <family>Noto Sans</family>
            <pointsize>20</pointsize>
           </font>
          </property>
          <property name="layoutDirection">
           <enum>Qt::RightToLeft</enum>
          </property>
          <property name="styleSheet">
           <string notr="true">QPushButton{
	background-color: white;
	border: 1px;
 	border-style: solid;
	border-color: rgb(255, 212, 246);
	color: #82005D;

 	border-radius:30px;
	max-width:60px;
	max-height:60px;
	min-width:60px;
	min-height:60px;
	

}</string>
          </property>
          <property name="text">
           <string>8</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="Button5">
          <property name="font">
           <font>
            <family>Noto Sans</family>
            <pointsize>20</pointsize>
           </font>
          </property>
          <property name="layoutDirection">
           <enum>Qt::RightToLeft</enum>
          </property>
          <property name="styleSheet">
           <string notr="true">QPushButton{
	background-color: white;
	border: 1px;
 	border-style: solid;
	border-color: rgb(255, 212, 246);
	color: #82005D;

 	border-radius:30px;
	max-width:60px;
	max-height:60px;
	min-width:60px;
	min-height:60px;
	

}</string>
          </property>
          <property name="text">
           <string>5</string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item>
       <layout class="QVBoxLayout" name="verticalLayout_3">
        <property name="spacing">
         <number>12</number>
        </property>
        <item>
         <widget class="QPushButton" name="ButtonPower">
          <property name="font">
           <font>
            <family>Noto Sans</family>
            <pointsize>20</pointsize>
           </font>
          </property>
          <property name="layoutDirection">
           <enum>Qt::RightToLeft</enum>
          </property>
          <property name="styleSheet">
           <string notr="true">QPushButton{
	background-color: #FF4BAC;
 	border-style: solid;
 	border-radius:30px;
	max-width:60px;
	max-height:60px;
	min-width:60px;
	min-height:60px;
	color: white;

}</string>
          </property>
          <property name="text">
           <string/>
          </property>
          <property name="icon">
           <iconset>
            <disabledon>:/icons/power_n.png</disabledon>
           </iconset>
          </property>
          <property name="iconSize">
           <size>
            <width>40</width>
            <height>30</height>
           </size>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="ButtonFact">
          <property name="font">
           <font>
            <family>Noto Sans</family>
            <pointsize>15</pointsize>
           </font>
          </property>
          <property name="layoutDirection">
           <enum>Qt::RightToLeft</enum>
          </property>
          <property name="styleSheet">
           <string notr="true">QPushButton{
	background-color: #FF4BAC;
 	border-style: solid;
 	border-radius:30px;
	max-width:60px;
	max-height:60px;
	min-width:60px;
	min-height:60px;
	color: white;

}</string>
          </property>
          <property name="text">
           <string>x!</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="Button7">
          <property name="font">
           <font>
            <family>Noto Sans</family>
            <pointsize>20</pointsize>
           </font>
          </property>
          <property name="layoutDirection">
           <enum>Qt::RightToLeft</enum>
          </property>
          <property name="styleSheet">
           <string notr="true">QPushButton{
	background-color: white;
	border: 1px;
 	border-style: solid;
	border-color: rgb(255, 212, 246);
	color: #82005D;

 	border-radius:30px;
	max-width:60px;
	max-height:60px;
	min-width:60px;
	min-height:60px;
	

}</string>
          </property>
          <property name="text">
           <string>7</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="Button4">
          <property name="font">
           <font>
            <family>Noto Sans</family>
            <pointsize>20</pointsize>
           </font>
          </property>
          <property name="layoutDirection">
           <enum>Qt::RightToLeft</enum>
          </property>
          <property name="styleSheet">
           <string notr="true">QPushButton{
	background-color: white;
	border: 1px;
 	border-style: solid;
	border-color: rgb(255, 212, 246);
	color: #82005D;

 	border-radius:30px;
	max-width:60px;
	max-height:60px;
	min-width:60px;
	min-height:60px;
	

}</string>
          </property>
          <property name="text">
           <string>4</string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
     </layout>
    </item>
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout">
      <property name="spacing">
       <number>12</number>
      </property>
      <property name="leftMargin">
       <number>23</number>
      </property>
      <property name="rightMargin">
       <number>23</number>
      </property>
      <item>
       <widget class="QPushButton" name="ButtonEqual">
        <property name="minimumSize">
         <size>
          <width>60</width>
          <height>132</height>
         </size>
        </property>
        <property name="font">
         <font>
          <family>Noto Sans</family>
          <pointsize>20</pointsize>
         </font>
        </property>
        <property name="cursor">
         <cursorShape>PointingHandCursor</cursorShape>
        </property>
        <property name="styleSheet">
         <string notr="true">QPushButton{

	background-color: qlineargradient(spread:pad, x1:0, y1:0, x2:0, y2:0.806818, stop:0 rgba(238, 34, 193, 255), stop:0.463054 rgba(248, 106, 216, 255), stop:0.852217 rgba(255, 164, 235, 255));
 	border-style: solid;
 	border-radius:30px;
	max-width:60px;
	max-height:132px;
	min-width:60px;
	min-height:132px;
	color: white;

}</string>
        </property>
        <property name="text">
         <string>=</string>
        </property>
       </widget>
      </item>
      <item>
       <layout class="QVBoxLayout" name="verticalLayout_2">
        <property name="spacing">
         <number>12</number>
        </property>
        <item>
         <layout class="QHBoxLayout" name="horizontalLayout_2">
          <property name="spacing">
           <number>12</number>
          </property>
          <item>
           <widget class="QPushButton" name="Button3">
            <property name="font">
             <font>
              <family>Noto Sans</family>
              <pointsize>20</pointsize>
             </font>
            </property>
            <property name="layoutDirection">
             <enum>Qt::RightToLeft</enum>
            </property>
            <property name="styleSheet">
             <string notr="true">QPushButton{
	background-color: white;
	border: 1px;
 	border-style: solid;
	border-color: rgb(255, 212, 246);
	color: #82005D;

 	border-radius:30px;
	max-width:60px;
	max-height:60px;
	min-width:60px;
	min-height:60px;
	

}</string>
            </property>
            <property name="text">
             <string>3</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QPushButton" name="Button2">
            <property name="font">
             <font>
              <family>Noto Sans</family>
              <pointsize>20</pointsize>
             </font>
            </property>
            <property name="layoutDirection">
             <enum>Qt::RightToLeft</enum>
            </property>
            <property name="styleSheet">
             <string notr="true">QPushButton{
	background-color: white;
	border: 1px;
 	border-style: solid;
	border-color: rgb(255, 212, 246);
	color: #82005D;

 	border-radius:30px;
	max-width:60px;
	max-height:60px;
	min-width:60px;
	min-height:60px;
	

}</string>
            </property>
            <property name="text">
             <string>2</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QPushButton" name="Button1">
            <property name="font">
             <font>
              <family>Noto Sans</family>
              <pointsize>20</pointsize>
             </font>
            </property>
            <property name="layoutDirection">
             <enum>Qt::RightToLeft</enum>
            </property>
            <property name="styleSheet">
             <string notr="true">QPushButton{
	background-color: white;
	border: 1px;
 	border-style: solid;
	border-color: rgb(255, 212, 246);
	color: #82005D;

 	border-radius:30px;
	max-width:60px;
	max-height:60px;
	min-width:60px;
	min-height:60px;
	

}</string>
            </property>
            <property name="text">
             <string>1</string>
            </property>
           </widget>
          </item>
         </layout>
        </item>
        <item>
         <layout class="QHBoxLayout" name="horizontalLayout_3">
          <property name="spacing">
           <number>12</number>
          </property>
          <item>
           <widget class="QPushButton" name="ButtonDot">
            <property name="font">
             <font>
              <family>Noto Sans</family>
              <pointsize>20</pointsize>
             </font>
            </property>
            <property name="layoutDirection">
             <enum>Qt::RightToLeft</enum>
            </property>
            <property name="styleSheet">
             <string notr="true">QPushButton{
	background-color: white;
	border: 1px;
 	border-style: solid;
	border-color: rgb(255, 212, 246);
	color: #82005D;

 	border-radius:30px;
	max-width:60px;
	max-height:60px;
	min-width:60px;
	min-height:60px;
	

}</string>
            </property>
            <property name="text">
             <string>·</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QPushButton" name="Button0">
            <property name="minimumSize">
             <size>
              <width>134</width>
              <height>62</height>
             </size>
            </property>
            <property name="maximumSize">
             <size>
              <width>134</width>
              <height>62</height>
             </size>
            </property>
            <property name="font">
             <font>
              <family>Noto Sans</family>
              <pointsize>20</pointsize>
             </font>
            </property>
            <property name="cursor">
             <cursorShape>PointingHandCursor</cursorShape>
            </property>
            <property name="mouseTracking">
             <bool>false</bool>
            </property>
            <property name="styleSheet">
             <string notr="true">QPushButton{
	background-color: white;
	border: 1px;
 	border-style: solid;
	border-color: rgb(255, 212, 246);
	color: #82005D;
 	border-radius:30px;
	max-width:112px;
	max-height:60px;
	min-width:112px;
	min-height:60px;
	padding-right: 20px;
	text-align:left;
	
}
</string>
            </property>
            <property name="text">
             <string>0</string>
            </property>
           </widget>
          </item>
         </layout>
        </item>
       </layout>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>350</width>
     <height>23</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
